presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestQueuesDb.java

```diff
@@ -307,20 +307,20 @@ public void testQueryTypeBasedSelection()
                 .build();
         QueryId queryId = createQuery(queryRunner, session, "EXPLAIN " + LONG_LASTING_QUERY);
         waitForQueryState(queryRunner, queryId, ImmutableSet.of(RUNNING, FINISHED));
-        Optional<String> resourceGroupName = queryRunner.getCoordinator().getQueryManager().getQueryInfo(queryId).getResourceGroupName();
-        assertTrue(resourceGroupName.isPresent(), "Query should have a resource group");
-        assertEquals(resourceGroupName.get(), "explain");
+        Optional<ResourceGroupId> resourceGroupId = queryRunner.getCoordinator().getQueryManager().getQueryInfo(queryId).getResourceGroupId();
+        assertTrue(resourceGroupId.isPresent(), "Query should have a resource group");
+        assertEquals(resourceGroupId.get(), new ResourceGroupId("explain"));
     }
 
     @Test
     public void testClientTagsBasedSelection()
             throws InterruptedException
     {
-        assertResourceGroupWithClientTags(ImmutableSet.of("tag1"), "global.bi-user");
-        assertResourceGroupWithClientTags(ImmutableSet.of("tag1", "tag2"), "global.user-user.adhoc-user");
+        assertResourceGroupWithClientTags(ImmutableSet.of("tag1"), new ResourceGroupId(ImmutableList.of("global", "bi-user")));
+        assertResourceGroupWithClientTags(ImmutableSet.of("tag1", "tag2"), new ResourceGroupId(ImmutableList.of("global", "user-user", "adhoc-user")));
```

Comment:
If there are tests where this is common you can create a helper (minor suggestion)